db/migrate/20240314140958_delete_specialist_topics_from_database.rb

```diff
@@ -0,0 +1,43 @@
+class DeleteSpecialistTopicsFromDatabase < ActiveRecord::Migration[7.1]
```

Comment:
As mentioned previously - I think it's safer to have a standalone PR for the database migration. And I'd definitely want to chat this PR through with someone from Publishing Platform first.